<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>747</width>
    <height>677</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>731</width>
      <height>621</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Kalendar">
     <attribute name="title">
      <string>Распределение дежурств</string>
     </attribute>
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>641</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame {
	border-radius: 10px;
	background-color: rgb(176, 224, 230);
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>401</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Futura LT</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Календарь для организации дежурств:</string>
       </property>
      </widget>
     </widget>
     <widget class="QCalendarWidget" name="calendar">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>321</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableWidget" name="tblwg_n_baddays">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>130</y>
        <width>361</width>
        <height>321</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Класс</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Кол-во дежурств</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Неудобный день</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>450</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Futura LT</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Выбранная дата:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ledit_date">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>490</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Futura LT</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_make_duty">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>490</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Futura LT</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Назначить</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ledit_class">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>490</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Futura LT</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>450</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Futura LT</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Дежурный класс:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_pick_class">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>540</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Futura LT</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Выбрать класс</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Личные данные</string>
     </attribute>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>-20</y>
        <width>721</width>
        <height>581</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame {
	border-radius: 10px;
	background-color: rgb(176, 224, 230);
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>70</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Futura LT</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Логин:</string>
       </property>
      </widget>
      <widget class="QLabel" name="lbl_register">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>20</y>
         <width>251</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Futura LT</family>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Личные данные:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>240</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Futura LT</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Пол:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="ledit_login">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>70</y>
         <width>221</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Futura LT</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </widget>
      <widget class="QLineEdit" name="ledit_surname">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>110</y>
         <width>221</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Futura LT</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>110</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Futura LT</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Фамилия:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>150</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Futura LT</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Имя:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="ledit_name">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>150</y>
         <width>221</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Futura LT</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>190</y>
         <width>191</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Futura LT</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Отчество:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="ledit_gender">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>240</y>
         <width>221</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Futura LT</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </widget>
      <widget class="QLineEdit" name="ledit_fathername">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>190</y>
         <width>221</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Futura LT</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Утвержденные дежурства</string>
     </attribute>
     <widget class="QTableWidget" name="tablewdt_dutys">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>140</y>
        <width>231</width>
        <height>351</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Дата</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Класс</string>
       </property>
      </column>
     </widget>
     <widget class="QFrame" name="frame_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>561</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame {
	border-radius: 10px;
	background-color: rgb(176, 224, 230);
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="lbl_register_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>541</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Futura LT</family>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Список утвержденных дежурств:</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>747</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
